src/CMakeLists.txt

```diff
@@ -615,6 +617,7 @@ install(PROGRAMS
 
 install(PROGRAMS
   ${CMAKE_SOURCE_DIR}/src/ceph-create-keys
+#  ${CMAKE_SOURCE_DIR}/src/ceph-disk
```

Comment:
This is a comment, it is not required to port this from https://github.com/starlingx-staging/stx-ceph/blob/ddfb3f76cc29f499fa13730e8ff6335d80cdc3b2/src/CMakeLists.txt#L964 but I think it is okay to leave it as it is.